Dayol-iOS/UI/DiaryPaper/DiaryPaperViewController.swift

```diff
@@ -88,7 +88,11 @@ class DiaryPaperViewController: DYViewController {
         stackView.alignment = .center
         return stackView
     }()
-    var contentsController = ContentsController()
+    lazy var contentsController: ContentsController = {
+        let scale: ScaleType = viewModel.orientation == .portrait ? .two : .three
+        let contentsController = ContentsController(scale: scale)
+        return contentsController
+    }()
```

Comment:
```suggestion
    lazy var contentsController: ContentsController = {
        let scale: ScaleType = viewModel.orientation == .two
        let contentsController = ContentsController(scale: scale)
        return contentsController
    }()
```

아직 가로속지 3배가 적용되지 않았습니다. TODO로 남겨놓고 2배로 해주세요